roles/runtime/defaults/main.yml

```diff
@@ -59,7 +59,12 @@ run__ml_tags:                       "{{ ml.tags | default(common__tags) }}"
 run__ml_public_loadbalancer:        "{{ ml.public_loadbalancer | default(run__public_endpoint_access) }}"
 
 run__dw_definitions:                "{{ dw.definitions | default([{}]) }}"
-run__dw_suffix:                     "{{ dw.suffix | default('dw') }}"
+run__dw_dbc_suffix:                 "{{ dw.dbc.suffix | default('dbc') }}"
+run__dw_vw_suffix:                  "{{ dw.vw.suffix | default('vw') }}"
+run__dw_tags:                       "{{ dw.tags | default(common__tags) }}"
+run__dw_overlay_network:            "{{ dw.overlay_network | bool }}"
+run__dw_private_load_balancer:      "{{ dw.private_load_balancer | default(not run__public_endpoint_access) }}"
```

Comment:
You are not passing `run__dw_private_load_balancer` and `run__dw_private_worker_nodes` fields to cloud module.

Reply:
Sorry, I messed up the rebase. Lines of code to pass these variables to the cluster module were dropped. Let me fix it.